rvzr/model_dynamorio/backend/include/speculator_abc.hpp

```diff
@@ -98,8 +98,8 @@ class SpeculatorABC
     /// @param type The type of the memory access (read or write)
     /// @param address The address of the memory access
     /// @param size The size of the memory access
-    /// @return void
-    virtual void handle_mem_access(bool is_write, void *address, uint64_t size);
+    /// @return false if the memory access cannot be rolled-back
```

Comment:
The phrasing is a bit confusing. Maybe "false if the memory access is invalid and is going to produce an exception"?